pkg/daemon/daemon.go

```diff
@@ -845,6 +850,9 @@ func (dn *Daemon) CheckStateOnBoot() error {
 		// All good!
 		return nil
 	}
+	if dn.recorder != nil {
+		dn.recorder.Eventf(getNodeRef(dn.node), corev1.EventTypeNormal, "BootResync", fmt.Sprintf("Booting node %s, currentConfig %s, desiredConfig %s", dn.node.Name, state.currentConfig.GetName(), state.desiredConfig.GetName()))
```

Comment:
Note that the name of this function is wrong, it should probably be `CheckStateOnMCDStart` or something, so this message would be `MCD starting on %s ...`